tree/ntuple/v7/inc/ROOT/RPageStorageDaos.hxx

```diff
@@ -33,7 +33,10 @@ namespace ROOT {
 namespace Experimental {
 
 namespace Internal {
-enum EDaosLocatorFlags { kNone = 0x00, kCaging = 0x01 };
+enum EDaosLocatorFlags {
+   // Indicates that the referenced page is "caged", i.e. it is stored in a larger blob that contains multiple pages.
+   kCagedPage = 0x01,
```

Comment:
We should create a new file under `tree/ntuple/v7/doc/` that explains the specifics of the DAOS backend, including the semantics of flags used in locators.
For a follow-up PR, though.